src/web/app/pages-instructor/instructor-session-result-page/instructor-session-no-response-panel.component.ts

```diff
@@ -57,6 +77,52 @@ export class InstructorSessionNoResponsePanelComponent implements OnInit, OnChan
     }
   }
 
+  openSendReminderModal(event: any): void {
```

Comment:
Let's make the request sent from the root page.

Reply:
I should have said `InstructorSessionResultPageComponent` (root page)

Reply:
All REST requests are moved to the root page.